database/load.py

```diff
@@ -1,28 +1,37 @@
 """Contains functions to upload scraped website data to a postgres database."""
+from time import perf_counter
```

Comment:
minor thing - but i think convention is to list the imports in alphabetical order. can clean up in all scripts closer to submission